### PR TITLE
Added function int16 to jax.numpy namespace

### LINKED ISSUE #7677: int16

### PR DESCRIPTION
**Pull request by D0m-inic**
Close #7677

### CODE REVIEW

**Comment by D0m-inic** [comment]
@abdrahmandiab please kindly review this PR when you have the time

File: ivy/functional/frontends/jax/lax/operators.py [hunk 1]
```diff
@@ -207,6 +207,11 @@ def gt(x, y):
     return ivy.greater(x, y)
 
 
+@to_ivy_arrays_and_back
+def int16(x):
+    return ivy.int16(x)
```
Comment (abdrahmandiab) [inline ivy/functional/frontends/jax/lax/operators.py:212]: This implementation is not the correct use of `ivy.int16()` 😅  [here](https://github.com/unifyai/ivy/actions/runs/3696647356/jobs/6260631786#step:6:1043).

Instead maybe try `ivy.astype(x , ivy.int16)`.

**Review by abdrahmandiab — CHANGES_REQUESTED** [review]
Hi @D0m-inic,
I've left a comment about some fixes that need to be made, let me know when they are done/ if you need help with them!

Thanks!

**Comment by D0m-inic** [comment]
Thanks for the comments, @abdrahmandiab, I have made changes if you could review when you have time

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by D0m-inic** [comment]
@abdrahmandiab please review, leaving a comment as it has been marked as stale.

**Comment by abdrahmandiab** [comment]
Hi @D0m-inic , apologies for the very late reply on this PR, I checked it out a bit earlier and completely forgot to give a review😅
I will review it by today or tomorrow hopefully, so please stay tuned for that! :)

**Review by abdrahmandiab — CHANGES_REQUESTED** [review]
Hi @D0m-inic, it seems the reason the test are failing is because there was a mistake in the directory in which you implemented the function & test 😅 

The correct location for it is under the [`jax.numpy.creation`](https://github.com/unifyai/ivy/blob/master/ivy/functional/frontends/jax/numpy/creation.py) category, rather than `jax.lax.operators`. You can also see the implementation for [uint16](https://github.com/unifyai/ivy/blob/96293b64abcf20eecde7e4a16fcea5aa4ecb85e8/ivy/functional/frontends/jax/numpy/creation.py#L67) and its [test](https://github.com/unifyai/ivy/blob/96293b64abcf20eecde7e4a16fcea5aa4ecb85e8/ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_creation.py#L286) for reference incase you face any more trouble! :)

Let me know if you need any help with this!

**Comment by D0m-inic** [comment]
Thanks @abdrahmandiab i've made changes, please review when you can.

**Comment by abdrahmandiab** [comment]
Hi @D0m-inic 
Everything looks great to me!
Fixed a tiny error with your implementation where you forgot to change the `fn_tree` from `jax.lax.int16` to `jax.numpy.int16`, and similarly with the test function's name. But otherwise all good!

Thanks!
